network/README.md

```diff
@@ -71,28 +68,33 @@ running as independent "tasks." The [tokio](https://tokio.rs/) framework is
 used as the task runtime. The different subcomponents in the network component
 are:
 
-* **NetworkProvider** &mdash; Exposes network API to clients. It forwards
-requests from upstream clients to appropriate downstream components and sends
+* **NetworkProvider** &mdash; An application level client interface to the network API.
+It forwards requests from upstream clients to appropriate downstream components and sends
 incoming RPC and DirectSend requests to appropriate upstream handlers.
-* **Peer Manager** &mdash; Listens for incoming connections and dials other
-peers on the network. It also notifies other components about new/lost
-connection events and demultiplexes incoming substreams to appropriate protocol
-handlers.
-* **Connectivity Manager** &mdash; Ensures that we remain connected to a node
-if and only if it is an eligible member of the network. Connectivity Manager
-receives addresses of peers from the Discovery component and issues
-dial/disconnect requests to the Peer Manager.
-* **Discovery** &mdash; Uses push-style gossip for discovering new peers and
-updates to addresses of existing peers. On every *tick*, it opens a new
-substream with a randomly selected peer and sends its view of the network to
-this peer. It informs the connectivity manager of any changes to the network
-detected from inbound discovery messages.
+
+* **Peer Manager** &mdash; Listens for incoming connections, and can dial outbound
+connections to other peers.  Demultiplexes and forwards messages to appropriate
+protocol handlers.  Additionally, notifies upstream components of new or closed
+connections.  Optionally can be connected to ConnectivityManager for a network with
+Discovery.
+
+* **Connectivity Manager** &mdash; Ensures connections to known peers found via
+Discovery.  Notifies Peer Manager to make outbound dials, or disconnects based
```

Comment:
Establishes? This isn't grammatically correct

Reply:
Yeah, Establishes is correct :P